zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -200,6 +203,7 @@ impl NetworkUpgrade {
         let spacing_seconds = match self {
             Genesis | BeforeOverwinter | Overwinter | Sapling => PRE_BLOSSOM_POW_TARGET_SPACING,
             Blossom | Heartwood | Canopy => POST_BLOSSOM_POW_TARGET_SPACING,
+            NU5 => unimplemented!("NU5 upgrade is still in progress and not fully implemented"),
```

Comment:
There are no proposed changes to block target spacing in NU5:
https://github.com/zcash/zips/pull/446/files#r583311733

```suggestion
            Blossom | Heartwood | Canopy | NU5 => POST_BLOSSOM_POW_TARGET_SPACING,
```